util/src/main/java/io/kubernetes/client/util/generic/options/CreateOptions.java

```diff
@@ -16,6 +16,11 @@ public class CreateOptions {
   private String dryRun;
   private String fieldManager;
 
+  public CreateOptions dryRun(String dryRun) {
```

Comment:
This change feels like it should be a different PR?

Reply:
I can do that... The reason I did this is that I was introducing methods to GetOptions and ListOptions and realized that none of them had builder-style methods and so I added them to all of the *Options classes.